style.css

```diff
@@ -312,7 +313,7 @@ button:active {
 .about_me {
   width: 100vw;
   max-width: var(--sections-max-width);
-  margin: 10% auto;
+  margin: auto;
```

Comment:
- [ ] I would kindly suggest you to work more on the About me section texts,must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)., make sure each texts to be align properly just like the figma design so that your website will be well professional and accessible by the users.

- [ ] Your design
![Screenshot 2023-06-10 031450](https://github.com/mateoburvi/Portfolio/assets/92458236/daadd4a2-89a9-4f82-97db-d95bb75b15ec)

- [ ] Figms design
![Screenshot 2023-06-10 030452](https://github.com/mateoburvi/Portfolio/assets/92458236/12195b3a-bd6c-48cc-a6ac-8af97298b76f)